packages/@webex/plugin-cc/test/unit/spec/services/agent/index.ts

```diff
@@ -0,0 +1,58 @@
+import routingAgent from '../../../../../src/services/agent';
+import AqmReqs from '../../../../../src/services/core/aqm-reqs';
+import * as Utils from '../../../../../src/services/core/Utils';
+
+jest.mock('../../../../../src/services/core/Utils', () => ({
+  createErrDetailsObject: jest.fn(),
+  getRoutingHost: jest.fn(),
+}));
+
+jest.mock('../../../../../src/services/core/aqm-reqs');
+
+describe('AQM routing agent', () => {
+  let fakeAqm: jest.Mocked<AqmReqs>;
+  let agent: ReturnType<typeof routingAgent>;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    const getroutingSpy = jest
```

Comment:
This is not required anymore I believe. Please remove it

Reply:
addressed